cmd/run.go

```diff
@@ -966,39 +1003,108 @@ func runStaticWorkload(cmd *cobra.Command, cacheType string, clientCount, rps in
 		cancel() // Cancel context to stop all workers
 	}()
 
-	// Create workers
+	// If using redis cache, proceed with normal setup. If using momento, pre-establish connections
+	// for accurate connection setup measurement before starting the workload.
```

Comment:
ellery and i chatted and we don't need to do this for momento, revert to the same behavior as redis where we just let each worker create its own client without the separation into 2 phases, sorry it's a pretty large change / revert